### PR TITLE
bypass original payload for kinesis publisher

### PR DESCRIPTION
**Pull request by semenodm**

### CODE REVIEW

File: src/main/scala/fs2/aws/kinesis/publisher.scala [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+              (command: Runnable) => ec.execute(command)
+            )
+          }
+      }
+
+    _.through(write)
+      .through(registerCallback)
+  }
+
+  def writeToKinesis1[F[_], I](
```
Comment (kiambogo) [inline src/main/scala/fs2/aws/kinesis/publisher.scala]: can we put some docs on this method?

Comment (kiambogo) [inline src/main/scala/fs2/aws/kinesis/publisher.scala]: perhaps `writeObjectToKinesis` is a better name?

**Comment by kiambogo** [comment]
add documentation and maybe rename method - up to you. otherwise LGTM

File: src/main/scala/fs2/aws/kinesis/RecordProcessor.scala [hunk 2]
```diff
@@ -14,7 +14,7 @@ import scala.collection.JavaConverters._
   *  @constructor create a new instance with a callback function to perform on record receive
   *  @param cb callback function to run on record receive, passing the new CommittableRecord
   */
-private[kinesis] class RecordProcessor(cb: CommittableRecord => Unit) extends v2.IRecordProcessor {
+private[aws] class RecordProcessor(cb: CommittableRecord => Unit) extends v2.IRecordProcessor {
```
Comment (kiambogo) [inline src/main/scala/fs2/aws/kinesis/RecordProcessor.scala]: why aws? this is a kinesis specific class

Comment (semenodm) [reply src/main/scala/fs2/aws/kinesis/RecordProcessor.scala]: i moved KinesisConsumerSpec to fs2.aws package and it is used there

Comment (semenodm) [reply src/main/scala/fs2/aws/kinesis/RecordProcessor.scala]: I can put it back to fs2.aws.kinesis and make it package private

Comment (semenodm) [reply src/main/scala/fs2/aws/kinesis/RecordProcessor.scala]: @dmateusp can you approve this PR?

**Pull request merged**